packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect, useCallback, useMemo } from 'react';
+import { ExtensionSlot, usePagination } from '@openmrs/esm-framework';
+import { RouteComponentProps } from 'react-router-dom';
+import styles from './patient-list-detail.scss';
+import { fetchPatientListDetails, OpenmrsCohort, fetchPatientListMembers, OpenmrsCohortMember } from '../api';
+import CustomOverflowMenuComponent from '../ui-components/overflow-menu.component';
+import { OverflowMenuVertical16 } from '@carbon/icons-react';
+import { useTranslation } from 'react-i18next';
+import { OverflowMenuItem, SkeletonText } from 'carbon-components-react';
+import PatientListTable from '../patient-table/patient-table.component';
+import dayjs from 'dayjs';
+
+interface PatientRow {
+  name: string;
+  identifier: string;
+  sex: string;
+  startDate: string;
+}
+
+function formatDate(date: string): string {
+  return dayjs(date).format('DD / MMM / YYYY');
+}
+
+interface PatientListDetailProps {
+  patientListUuid: string;
+}
+
+const PatientListDetailComponent: React.FC<RouteComponentProps<PatientListDetailProps>> = ({ match }) => {
+  const [patientListDetails, setPatientListDetails] = useState<OpenmrsCohort>(null);
+  const [patientListMembers, setPatientListMembers] = useState<OpenmrsCohortMember[]>([]);
+  const [isLoading, setLoading] = useState(true);
+  const { patientListUuid } = match.params;
+  const { t } = useTranslation();
+  const [currentPageSize, setCurrentPageSize] = useState(10);
+  const [searchString, setSearchString] = useState('');
+
+  useEffect(() => {
+    const abortController = new AbortController();
+    fetchPatientListDetails(patientListUuid, abortController).then((res) => setPatientListDetails(res.data));
+    fetchPatientListMembers(patientListUuid, abortController).then((res) => {
+      setPatientListMembers(res);
+      setLoading(false);
+    });
+    return () => {
+      abortController.abort();
+    };
+  }, []);
+
+  const headers = useMemo(
+    () => [
```

Comment:
The `header` properties below should have translations (keys and their corresponding translation strings).